config/locales/flood_risk_engine/_local_authority.en.yml

```diff
@@ -0,0 +1,14 @@
+---
+en:
+  flood_risk_engine:
+    enrollments:
+      steps:
+        local_authority: 
+          heading: What's the name of the local authority or public body?
+          label: Local authority or public body name
+          legend: Local authority or public body name
+          para_text: For example, Bristol City Council
+          errors:
+            name:
+             blank: Enter the name of the local authority or public body
+             too_long: Check the name - it should have no more than than 200 characters
```

Comment:
The down side is that ActiveModel doesn't know where to find the error texts, so you have to specify the whole path to the message. If you have a section:
```
  activemodel:
    errors:
      messages:
        some_error: Some error message
```
You can then do:
```
validates(
        :name,
        presence: {
          message: I18n.t("some_error")
        }
      )
```
I guess that could live in another part of the other locale data for the page/form.